src/Symfony/Component/HttpClient/Tests/CurlConditionalHttpClientTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpClient\Tests;
+
+use Symfony\Component\HttpClient\ConditionalHttpClient;
+use Symfony\Component\HttpClient\CurlHttpClient;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+use Symfony\Contracts\HttpClient\Test\HttpClientTestCase;
+
+/**
+ * @requires extension curl
+ */
+class CurlConditionalHttpClientTest extends HttpClientTestCase
```

Comment:
tests should not extend `HttpClientTestCase` as we don't need to test the wrapped client
we need a mock client instead that verifies when/what it is called with